CONTRIBUTING.md

```diff
@@ -18,10 +26,103 @@ By contributing to Backstage Community Plugins you agree to adhere to the [CNCF
 
 [code-of-conduct]: https://github.com/cncf/foundation/blob/main/code-of-conduct.md
 
+## License
+
+The community plugins repository is under [Apache 2.0](../LICENSE) license. All plugins added & moved to the repository will be kept under the same license. If you are moving a plugin over make sure that no other license file is in the plugin workspace & all `package.json` files either have no version defined or explicitly use _“Apache 2.0”_.
+
 ## Security Issues
 
 See [SECURITY](SECURITY.md).
 
+## Get Started!
+
+So...feel ready to jump in? Let's do this. 👏🏻 💯
+
+### Cloning the Repository
+
+Ok. So you're gonna want some code right? Go ahead and fork the repository into your own GitHub account and clone that code to your local machine. GitHub's [Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo) documentation has a great step by step guide if you are not sure how to do this.
+
+If you cloned a fork, you can add the upstream dependency like so:
+
+```bash
+git remote add upstream git@github.com:backstage/community-plugins.git
+git pull upstream main
+```
+
+After you have cloned the Community Plugins repository, you should run the following commands once to set things up for development:
+
+```bash
+# jump in to the community-plugins repo that you cloned
+cd community-plugins
+# install the root dependencies so that you can create workspaces if needed
+yarn install
+# navigate to a workspace that you're working on
+cd workspaces/linguist
+# install the workspace dependencies
+yarn install
+```
+
+### Developing Plugins in Workspaces
+
+Frontend and Backend plugins come with a standalone runner that you should be able to utilize in order to develop on your plugins in isolation. You can navigate to a workspace and a plugin inside the plugin folder and run `yarn start` which should kick off the development standalone server for that plugin.
```

Comment:
Do we want to call out that this stand alone aspect is a bit of work in progress? I would bet that most of the plugins don't work properly this way or the ones that do don't really help work with the plugin during local development.

Reply:
Example for the Explore plugin from Discord: https://discord.com/channels/687207715902193673/1232681492702826558/1233013371310444565

And I'm sure the Azure DevOps and Linguist plugins would fail as well :(

Reply:
Yeah, I've added a note about them maybe not being setup. I'm going to create a followup issue so that we can track which workspaces have been setup also, and maybe get some good first issue contributions on it.

The statement here was basically made for reference on what we should do for local dev for new plugins,. not really specifically chatting about the migrated plugins per se.

Reply:
https://github.com/backstage/community-plugins/issues/413